lib/meteor-analytics.js

```diff
@@ -4,37 +4,51 @@
 * in the database when using oAuth login.
 */
 getUserEmail = function(){
-  if ( Meteor.user().emails ) {
-    if (Meteor.user().emails[0]) {
-      return Meteor.user().emails[0].address;
-    } else {
-      return null;
-    }
-  } else if ( Meteor.user().services ) {
-    var services = Meteor.user().services;
-    if ( services.facebook ) {
-      return services.facebook.email;
-    } else if ( services.github ) {
-      return services.github.email;
-    } else if ( services.google ) {
-      return services.google.email;
-    } else {
-      return null;
+  if (Meteor.userId()) {
```

Comment:
@pauldowman, I was not able to get rid of this line completely, but I did change it to use `userId` and that allows the update only on email field changes.